examples/single-source-advanced/README.md

```diff
@@ -9,61 +9,67 @@ To get familiarized, we will show how you can use Secret Sync tool to cover two
 
 *Note:* The same logic applies to any other combination of secret stores.
 
-### Step 1: Prepare environment
+## Step 1: Prepare environment
 
 You will need the following tools to continue:
+
+- Docker compose
 - Git
 - Makefile
 - Golang `>= 1.21`
 
 To set up the environment, you can build from source:
 
 ```bash
-git clone https://github.com/bank-vaults/secret-sync.git /tmp/secret-sync
-cd /tmp/secret-sync
+git clone https://github.com/bank-vaults/secret-sync.git tmp/secret-sync
+cd tmp/secret-sync
```

Comment:
please use absolute paths otherwise the examples wont work properly in certain cases
```suggestion
git clone https://github.com/bank-vaults/secret-sync.git /tmp/secret-sync
cd /tmp/secret-sync
```

Reply:
this applies to both examples